plugins/rest/rest.go

```diff
@@ -157,43 +100,16 @@ func New(config []byte, opts ...func(*Client)) (Client, error) {
 		return Client{}, err
 	}
 
-	tlsConfig, err := parsedConfig.validateAndInjectDefaults()
+	parsedConfig.URL = strings.TrimRight(parsedConfig.URL, "/")
```

Comment:
Why do we have to trim the URL? Is this something that should always be done or plugin specific?

Reply:
I wasn't sure exactly what that was meant for, but I didn't want to change that as I was worried about introducing a regression. It used to be in validateAndInjectDefaults: 
https://github.com/open-policy-agent/opa/blob/3263f54a74242f2225d0d7520f401805970c86db/plugins/rest/rest.go#L51

Reply:
Oh interesting, I missed that we were already doing it before 😄 

It is kinda weird but I do agree we shouldn't change the behavior.